client/App.jsx

```diff
@@ -120,8 +120,9 @@ class App extends Component {
     if (this.props.watch.length && this.props.watch[0] === term) {
       return;
     }
-
+if (isTX(term) && isBlock(term) && isAddress(term)) {
```

Comment:
I think we need OR statements here or the check will require that the term is a tx, block, and address.
```
if (isTX(term) || isBlock(term) || isAddress(term)) {
```